Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -680,8 +794,12 @@ public function setCode($code)
             $this[$index] = new Token($token);
         }
 
+        $transformers = Transformers::create();
+        $transformers->transform($this);
```

Comment:
maybe even 1.4 if the transformers are already in it (I don't remember what got in 1.4)